package.json

```diff
@@ -65,19 +108,24 @@
     "react-redux": "^5.0.6",
     "redux": "^4.0.0",
     "tslint": "^5.8.0",
+    "ts-loader": "^4.4.2",
+    "ts-jest": "^23.0.1",
     "tslint-config-standard": "^7.0.0",
     "tslint-react": "^3.2.0",
-    "ts-loader": "^4.4.2",
     "typescript": "^2.8.1",
     "url-loader": "^0.6.2",
     "webpack": "^4.16.0",
     "webpack-cli": "^3.0.8"
   },
   "dependencies": {
+    "@types/enzyme-adapter-react-16": "^1.0.2",
```

Comment:
Please move it into devDep

Reply:
just moved it all under dev dependencies